documentation/synthetic_tokens/tutorials/liquidation.md

```diff
@@ -0,0 +1,423 @@
+
+Alternatively if you want to push a new local config file you can do this by running the following command (assuming we are updating the `ethbtc-liquidator-kovan` initialized in the previous code block)
+
+```bash
+ gcloud compute instances update-container ethbtc-liquidator-kovan \
+        --container-image docker.io/umaprotocol/protocol:latest \
+        --container-env-file ./liquidator.env
+```
+
+## Bot health monitoring
+
+Once your bots are running on GCP you want to know when they are properly executing liquidations or if any errors occur. The bots by default provide a few useful monitoring avenues. These are now discussed.
+
+### GCP StackDriver intergration
+
+Once your containers are running within compute engine you properly want to see the log messages in the same way you did when running `docker attach` in the previous section.
+To make this process easier the bot logger configuration includes Google Stackdriver integration.
+This enables GCP logging to pick up the log messages generated from the bots. To enable this add the following environment variable to the docker containers:
+
+```bash
+ENVIRONMENT=production
+```
+
+This will tell the bot's logger to pipe logs to GCP logging. For more information on this GCP logging and how to access it within Gcloud see the official doc [here](https://cloud.google.com/logging).
+
+### Slack integration
+
+The bots by default also come with built in slack integration to send slack messages when events occur (bot liquidates a position, is running low on capital or something has failed).
+To use this integration you must first generate a Slack webhook.
+Information on generating a webhook can be found on the slack docs [here](https://slack.com/intl/en-za/help/articles/115005265063-Incoming-Webhooks-for-Slack).
+
+Once you've set up a Slack webhook you can add it to the bots by adding the following environment variable:
+
+```bash
+SLACK_WEBHOOK=<your slack webhook>
+```
+
+The bots will now automatically start sending log messages in Slack.
+
+## Running a liquidator bot on mainnet
+
+The tutorial thus far assumed you are running a Kovan liquidator and dispute bots. Next, we will discuss how to move the deployment onto the Main Ethereum network. This involves three main steps which are outlined below.
+
+**1) Funding your liquidator bot's main net wallet**
+
+Run on the Mainnet involves first repeating the [Funding accounts](#Funding-accounts) section on the main Ethereum network to acquire collateral to fund the liquidator and disputer bots.
+
+**2) Updating the EMP address to point to main net expiring multi party contract**
+
+Update your environment configuration `EMP_ADDRESS` to refer to the mainnet address of the expiring multiparty contract you want to monitor.
+
+<!-- TODO: add a link to another docs page that outlines the EMP address for all main net deployments -->
+
+**3) Update the `COMMAND` used to start the bots to point at the mainnet, rather than kovan.**
```

Comment:
nit: for instance, we use `mainnet` here rather than `main net`.

Reply:
bump

Reply:
have changed all to mainnet